README.md

```diff
@@ -6,6 +6,51 @@ The official [Helm chart](https://helm.sh/) for [Hypeledger Firefly](https://hyp
 related connector microservices. See the [chart `README`](charts/firefly/README.md) for installation and
 configuration instructions.
 
+## Quick Start
+
+If you want to run these charts locally on your own machine, you can run a single command to get a fully working stack, end-to-end:
+
+```
+make stack
+```
+
+This will create a pre-set environment with the following configuration:
+
+- Runs all containers in [kind](https://kind.sigs.k8s.io/)
+- Sets up a PostgreSQL DB in the K8s cluster
+- Creates a basic single node Besu blockchain also running in the K8s cluster
+- Sets up FireFly and all of its dependencies to use these services
+- Sets up an ERC-20 / ERC-721 Token Connector in this stack
+- Provides an optional script to enable multiparty mode after initial set up
+
+If you wish to make changes to your stack you can modify `./charts/firefly/local-kind-values.yaml` and run:
+
+```
+helm upgrade --install firefly ./charts/firefly -f ./charts/firefly/local-kind-values.yaml
+```
+
+### Enabling multiparty mode
+
+After you run the quickstart command above, you can also (optionally) enable Multiparty mode. This will enabled FireFly's advanced Messaging features. To enable that, you can run the shell script:
+
+```
+./hack/multiparty.sh
```

Comment:
Does this work with the new APIs we have introduced in main?

Reply:
If not could you please raise an issue as a follow up

Reply:
See https://github.com/hyperledger/firefly-helm-charts/pull/76#issuecomment-2186095649